.github/workflows/go.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - main
-      - 'feat-**'
```

Comment:
one small question: will that deletion affect PR CI of branches with prefix "feat-"?

Reply:
Yes, we did this in v2, it was duplicating checks.

Reply:
nit: yah why this?